openeo/files.js

```diff
@@ -119,6 +128,12 @@ var Files = {
 				}
 				else {
 					res.send(200);
+					const payload = {
+						user_id: req.user._id,
+						path: p.replace('storage/user_files/'+req.user._id+'/', ''),
+						action: 'deleted'
+					};
+					this.subscriptions.broadcast('openeo.files', {}, payload); // TODO: params shouldn't be empty object but the payload, so that clients can subscribe to file-specific messages etc.
```

Comment:
Hmm... we should discuss whether this should be a broadcast. Should file subscriptions be user-specific for now?

Also, maybe we should just not yet mess around with file-specific subscriptions, but only allow subscribing to the whole user-space for now.

Reply:
I think file-specific subscriptions aren't that hard to implement and would be a nice feature. However I'll admit that it's a rather specific feature that we can care about later. Does it hurt if we leave the "TODO" here?

Reply:
issue written :heavy_check_mark:
see :arrow_right: openeo-api#119